ci/do_ci.sh

```diff
@@ -92,6 +92,22 @@ elif [[ "$1" == "cmake.legacy.test" ]]; then
   make
   make test
   exit 0
+elif [[ "$1" == "cmake.legacy.exporter.otprotocol.test" ]]; then
+  cd "${BUILD_DIR}"
+  rm -rf *
+  export BUILD_ROOT="${BUILD_DIR}"
+  ${SRC_DIR}/tools/build-gtest.sh
+  ${SRC_DIR}/tools/build-benchmark.sh
+  cmake -DCMAKE_BUILD_TYPE=Debug  \
+        -DCMAKE_CXX_STANDARD=11 \
+        -DWITH_OTLP=ON \
```

Comment:
Do you think we could set `WITH_OTLP` to `ON` by default?

Reply:
You mean make it default here:
https://github.com/open-telemetry/opentelemetry-cpp/blob/ef16d0048a0ccd6d664b86da1c95294633965bad/CMakeLists.txt#L144

I will raise an issue for this, and we can handle it separately as this is not related to this PR?

Reply:
Sounds good.